keyboards/th96/keymaps/default/keymap.c

```diff
@@ -0,0 +1,109 @@
+		KC_NO,    KC_NO,  KC_NO,  KC_NO,  KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_NO,   KC_NO,  KC_NO,              KC_NO,    KC_NO,
+		KC_NO,    KC_MPLY,KC_MPRV,KC_MNXT,KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_NO,   KC_NO,  KC_NO,              KC_NO,    KC_NO,    KC_NO,    KC_NO,
+		KC_NO,    KC_MSTP,KC_NO,  KC_NO,  KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_NO,   KC_NO,                      KC_NO,    KC_NO,    KC_NO,    KC_NO,
+		KC_TRNS,  KC_NO,  KC_NO,  KC_NO,  KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_TRNS, RGB_VAI,                    KC_NO,    KC_NO,    KC_NO,
+		KC_NO,    KC_NO,  KC_NO,                        KC_NO,                   MO(3),   KC_TRNS,RGB_SPI, RGB_VAD,RGB_SPD,                      KC_NO,    KC_NO,    KC_NO
+    ),
+
+    [2] = LAYOUT_all(
+	    RGB_TOG,  RGB_MOD,RGB_HUI,RGB_SAI,KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_NO,   KC_NO,  KC_NO,              KC_NO,    KC_NO,    KC_TRNS,
+		KC_NO,    KC_NO,  KC_NO,  KC_NO,  KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_NO,   KC_NO,  KC_NO,              KC_NUM,   KC_NO,
+		KC_NO,    KC_MPLY,KC_MPRV,KC_MNXT,KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_NO,   KC_NO,  KC_NO,              KC_NO,    KC_NO,    KC_NO,    KC_NO,
+		KC_NO,    KC_MSTP,KC_NO,  KC_NO,  KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_NO,   KC_NO,                      KC_NO,    KC_NO,    KC_NO,    KC_NO,
+		KC_TRNS,  KC_NO,  KC_NO,  KC_NO,  KC_NO, KC_NO, KC_NO,  KC_NO,  KC_NO,   KC_NO,   KC_NO,  KC_TRNS, RGB_VAI,                    KC_NO,    KC_NO,    KC_NO,
+		KC_NO,    KC_NO,  KC_NO,                        KC_NO,                   MO(3),   KC_TRNS,RGB_SPI, RGB_VAD,RGB_SPD,                      KC_NO,    KC_NO,    KC_NO
+    ),
+
+    [3] = LAYOUT_all(
+		RGB_TOG,RGB_MOD,RGB_HUI,RGB_SAI,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,    KC_NO,             KC_NO,    KC_NO,    KC_TRNS,
+		QK_BOOT,KC_NO,  KC_NO,  KC_NO,    KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,    KC_NO,             KC_NO,    KC_NO,
+		KC_NO,  KC_NO,  KC_NO,  KC_NO,    KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,    KC_NO,             KC_NO,    KC_NO,    KC_NO,    KC_NO,
+		KC_NO,  KC_NO,  KC_NO,  KC_NO,    KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,                       KC_NO,    KC_NO,    KC_NO,    KC_NO,
+		KC_NO,  KC_NO,  KC_NO,  KC_NO,    KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  RGB_VAI,                     KC_NO,    KC_NO,    KC_NO,
+		KC_NO,  KC_NO,  KC_NO,                            KC_NO,                  KC_TRNS,KC_TRNS,RGB_SPI,RGB_VAD,  RGB_SPD,                     KC_NO,    KC_NO,    KC_NO
+    )
+
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
```

Comment:
Please convert to GPIO commands: 
https://docs.qmk.fm/#/gpio_control?id=functions